src/dialects/sqlite/query-generator-typescript.ts

```diff
@@ -48,4 +48,21 @@ export class SqliteQueryGeneratorTypeScript extends MySqlQueryGenerator {
       this.quoteIdentifier(indexName),
     ]);
   }
+
+  /**
+   * @override
+   */
+  getForeignKeyQuery(_tableName: TableNameOrModel, _columnName: string): string {
+    throw new Error(`getForeignKeyQuery has not been implemented in ${this.dialect.name}.`);
```

Comment:
I think this error should say "is not supported" instead of not implemented (unless it's possible to implement and is a TODO)